docs/policy-packages.md

```diff
@@ -50,24 +50,29 @@ fallback can be implemented with code similar to the following:
     if action not in perm:
         return rucio.core.permission.generic.has_permission(issuer, action, kwargs)
 
-init.py should include a
-SUPPORTED_VERSION field indicating the version of Rucio
+`__init__.py` should include a
+`SUPPORTED_VERSION` field indicating the version of Rucio
 that your package was developed against. This may be checked by Rucio in
 the event that the policy package interface changes in the future.
 Example:
 
     SUPPORTED_VERSION = "1.20.7"
 
-Custom surl construction algorithms can be registered in
-`init.py`:
+It can also contain an optional function called `get_surl_algorithms` that
+returns a dictionary of custom surl construction algorithms implemented
+within the package:
 
-    from rucio.common.utils import register_surl_algorithm
-    register_surl_algorithm(construct_surl_special, 'voname_special')
+    def get_surl_algorithms():
+        return { 'voname_special': construct_surl_special }
 
-So can custom lfn to pfn algorithms:
+where `construct_surl_special` is the function within the policy package
+that implements the algorithm.
 
-    from rucio.rse.protocols.protocol import RSEDeterministicTranslation
-    RSEDeterministicTranslation.register(lfn2pfn_special, 'voname_special')
+Another optional function called `get_lfn2pfn_algorithms` does the same
+for lfn to pfn translation algorithms:
+
+    def get_lfn2pfn_algorithms():
+        return { 'voname_special': lfn2pfn_special }
```

Comment:
it would make sense to enclose (python) code in Markdown like this:

\`\`\`py
```py
def get_lfn2pfn_algorithms():
    return { 'voname_special': lfn2pfn_special }
```
\`\`\`

(same above)

Reply:
Thanks, I have updated the markdown to use the Python code blocks. I meant to say, it may be best not to merge this until https://github.com/rucio/rucio/pull/4667 is merged into the main repo to implement the changes that are documented here.